server/fleet/activities.go

```diff
@@ -1283,17 +1298,22 @@ func (a ActivityTypeUnlockedHost) Documentation() (activity, details, detailsExa
 }`
 }
 
-func (a ActivityTypeEditedWindowsProfile) ActivityName() string {
-	return "edited_windows_profile"
+type ActivityTypeWipedHost struct {
```

Comment:
Just some reordering as the Documentation of the EditedWindowsProfile activity was away from the rest of this type's methods.